wire-runtime-swift/src/main/swift/Codable/StringEncoded+Data.swift

```diff
@@ -16,26 +16,26 @@
 
 import Foundation
 
-extension Data : StringCodable {
+extension Foundation.Data : StringCodable {
```

Comment:
We shouldn't need to modify anything in `wire-runtime-swift/src/main/swift`

Reply:
@lickel not sure why I recall seeing an error. Either way this is reverted and the build still passes.